### PR TITLE
Update Drashna user space code to support keyboard_init functions

### PR DESCRIPTION
**Pull request by drashna**

### CODE REVIEW

File: users/drashna/rgb_stuff.c [hunk 1]
```diff
@@ -336,6 +336,7 @@ uint32_t layer_state_set_rgb(uint32_t state) {
   return state;
 }
 
+#if 0
```
Comment (mechmerlin) [inline users/drashna/rgb_stuff.c]: Doing this means this is not even compiled. What was the purpose of this?

Comment (drashna) [reply users/drashna/rgb_stuff.c]: It's old code, it works, but ... it's no longer needed with the `keyboard_post_init_rgb` function.  But I'd rather keep it here, inc case it's useful in the future.

Comment (mechmerlin) [reply users/drashna/rgb_stuff.c]: Perhaps you should actually put a comment regarding that here. You're fairly well known, enough for people to copy and paste your code.

Comment (drashna) [reply users/drashna/rgb_stuff.c]: Done

**Pull request merged**